### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by super-cooper**
Update the issue templates to use the correct labels

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/improvement.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 name: Improvement
```
Comment (nhawke) [inline .github/ISSUE_TEMPLATE/improvement.md:2]: Unclear to me why we need 2 separate templates for this. an "Improvement" is just a Feature Request. we could probably just remove this template altogether

Comment (super-cooper) [reply .github/ISSUE_TEMPLATE/improvement.md:2]: New feature vs improving existing feature

Comment (nhawke) [reply .github/ISSUE_TEMPLATE/improvement.md:2]: yes i know. the templates are practically idential though and semantically nearly identical. seems redundant to have

Comment (super-cooper) [reply .github/ISSUE_TEMPLATE/improvement.md]: ok

**Review by nhawke — APPROVED** [review]

**Pull request merged**